examples/Workflow/WorkflowWebApp/Program.cs

```diff
@@ -1,51 +1,78 @@
-﻿using Dapr.Workflow;
+﻿using System.Text.Json.Serialization;
+using Dapr.Workflow;
+using Microsoft.AspNetCore.Mvc;
+using WorkflowWebApp.Activities;
+using WorkflowWebApp.Workflows;
+using JsonOptions = Microsoft.AspNetCore.Http.Json.JsonOptions;
 
 // The workflow host is a background service that connects to the sidecar over gRPC
 WebApplicationBuilder builder = WebApplication.CreateBuilder(args);
 
+// Configure HTTP JSON options.
+builder.Services.Configure<JsonOptions>(options =>
+{
+    options.SerializerOptions.Converters.Add(new JsonStringEnumConverter());
+    options.SerializerOptions.DefaultIgnoreCondition = JsonIgnoreCondition.WhenWritingNull;
+});
+
 // Dapr workflows are registered as part of the service configuration
 builder.Services.AddDaprWorkflow(options =>
 {
-    // Example of registering a "PlaceOrder" workflow function
-    options.RegisterWorkflow<string, string>("PlaceOrder", implementation: async (context, input) =>
-    {
-        // In real life there are other steps related to placing an order, like reserving
-        // inventory and charging the customer credit card etc. But let's keep it simple ;)
-        return await context.CallActivityAsync<string>("ShipProduct", "Coffee Beans");
-    });
+    options.RegisterWorkflow<OrderProcessingWorkflow>();
```

Comment:
Nice! It might be worth commenting that one _can_ define workflows and activities inline _but_ that in this example we define them in their own classesmore organized but that doing so isn't required...

Reply:
Agreed. I'll add a comment.